seed/seedwriter/writer.go

```diff
@@ -1423,13 +1423,29 @@ func (w *Writer) SeedSnaps(copySnap func(name, src, dst string) error) error {
 	return nil
 }
 
+func (w *Writer) markValidationSetsSeeded() error {
+	vsm, err := w.validationSetAsserts()
+	if err != nil {
+		return err
+	}
+	for seq, vs := range vsm {
+		w.manifest.MarkValidationSetSeeded(vs, seq.Pinned)
```

Comment:
Function return error is ignored.

Reply:
Nice catch, that was not intentional. Added the error check.